pkg/hive/hive.go

```diff
@@ -146,21 +146,23 @@ func (h *Hive) Viper() *viper.Viper {
 type defaults struct {
 	dig.Out
 
-	Flags       *pflag.FlagSet
-	Lifecycle   Lifecycle
-	Logger      logrus.FieldLogger
-	Shutdowner  Shutdowner
-	InvokerList cell.InvokerList
+	Flags             *pflag.FlagSet
+	Lifecycle         Lifecycle
+	Logger            logrus.FieldLogger
+	Shutdowner        Shutdowner
+	InvokerList       cell.InvokerList
+	EmptyFullModuleID cell.FullModuleID
 }
 
 func (h *Hive) provideDefaults() error {
 	return h.container.Provide(func() defaults {
 		return defaults{
-			Flags:       h.flags,
-			Lifecycle:   h.lifecycle,
-			Logger:      log,
-			Shutdowner:  h,
-			InvokerList: h,
+			Flags:             h.flags,
+			Lifecycle:         h.lifecycle,
+			Logger:            log,
+			Shutdowner:        h,
+			InvokerList:       h,
+			EmptyFullModuleID: cell.FullModuleID(""),
```

Comment:
nit: Perhaps define a const BlankModuleID if deemed useful?

Reply:
Changed `FullModuleID` to be `[]string` underneath, so now it's just `nil`.